go/exec/vitess_config.go

```diff
@@ -0,0 +1,124 @@
+/*
+ *
+ * Copyright 2022 The Vitess Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * /
+ */
+
+package exec
+
+import (
+	"fmt"
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/vitessio/arewefastyet/go/infra/ansible"
+	"github.com/vitessio/arewefastyet/go/tools/git"
+)
+
+type (
+	vitessConfig struct {
+		vtgate, vttablet string
+	}
+
+	rawVitessVersionConfig struct {
+		version git.Version
+		value   map[string]interface{}
+	}
+)
+
+func (e *Exec) prepareVitessConfiguration() error {
+	if len(e.rawVitessConfig) == 0 {
+		return nil
+	}
```

Comment:
adding a test for this file will be great to see if this works as expected.

Reply:
done